src/main/java/com/meilisearch/sdk/TenantTokenOptions.java

```diff
@@ -0,0 +1,28 @@
+package com.meilisearch.sdk;
+
+import java.util.Date;
+
+/** The option you want to pass for generate a Tenant Token */
+public class TenantTokenOptions {
+
+    public TenantTokenOptions() {}
+
+    protected String apiKey = null;
+    protected Date expiresAt = null;
+
+    public String getApiKey() {
+        return apiKey;
+    }
+
+    public void setApiKey(String apiKey) {
+        this.apiKey = apiKey;
+    }
+
+    public Date getExpiresAt() {
```

Comment:
This `Date` can handle time as well?

Reply:
What do you mean?

Reply:
If you want to deal with `Date` in ruby you can use [`Time`](https://ruby-doc.org/core-3.1.0/Time.html) or [`Date`](https://ruby-doc.org/stdlib-3.1.1/libdoc/date/rdoc/Date.html), but if you want to deal with both you cannot use `Date`. My question is related to the `time` part of a date and if this Date class behaves like Ruby's or not 😄

Reply:
It could be quite similar because the Class `Date` encapsulates the `Date` and `Time` but the `Time` is just used for (hour, minutes, second, millisecond ...) I'm not really sure you can use `Time` for a `Date` if you need a `Date`.